gems/activerecord/6.0/activerecord-6.0.rbs

```diff
@@ -1,4 +1,10 @@
 module ActiveRecord
+  class Base
+    module ClassMethods[Model, Relation, PrimaryKey]
+      def create_with: (nil | Hash[untyped, untyped]) -> Relation
```

Comment:
Why do you add this to each version? It seems all definitions are the same. I feel it's enough to add this to `activerecord.rbs`.

Reply:
Thanks for your comment!

I didn't know that if you write in activerecord.rbs, the rbs is shared in each version.
I rewrote create_with rbs definition in activerecord.rbs.